source/clear-linux/tutorials/fmv.rst

```diff
@@ -186,7 +186,7 @@ verify, enter the following commands:
     vpaddd (%r8,%rax,1),%ymm0,%ymm0
     vmovdqu %ymm0,(%rcx,%rax,1)
 
-FTT project example
+FFT project example
```

Comment:
Thanks for correcting the typo.  The example below this shows use of FFTW, not FFT. I'd like clarification on this.  
http://www.fftw.org/faq/section1.html#whatisfftw 
@pixelgeek , @phmccarty, @mrkz can one of you help determine which is correct here, FFT or FFTW?

Reply:
FFTW is the project name, and FFT is an algorithm, so I would replace all mention of FFT in this tutorial with FFTW.

Reply:
Hello @pixelgeek. Will you please make edits per comments from @phmccarty?

Reply:
Updated - some are generic FFT references, some are FFTW library code references.